apps/web/src/views/GaugesVoting/components/Table/VoteTable/hooks/useUrlQueryGauge.ts

```diff
@@ -0,0 +1,74 @@
+import { Gauge } from '@pancakeswap/gauges'
+import { useTranslation } from '@pancakeswap/localization'
+import { useToast } from '@pancakeswap/uikit'
+import { watchAccount } from '@wagmi/core'
+import useAccountActiveChain from 'hooks/useAccountActiveChain'
+import { useSearchParams } from 'next/navigation'
```

Comment:
I think this hooks is for app router not page router?

Reply:
🤔 I think next eslint config should add a this config, 

warning all app router api if use page router